src/frontend/apps/lti_site/components/Transcripts/index.tsx

```diff
@@ -88,47 +102,83 @@ export const Transcripts = ({ transcripts }: TranscriptsProps) => {
+
   return (
-    <React.Fragment>
-      <Box align="center" justify="center" direction="row" pad="medium">
-        <form>
-          <LabelChooseLanguage htmlFor="languages">
-            <FormattedMessage {...messages.transcriptLabel} />
-          </LabelChooseLanguage>
-          <select
-            id="languages"
-            name="language_choices"
-            onChange={onSelectChange}
-            value={selected.language}
+    <Box
+      background={{ color: 'bg-marsha' }}
+      style={{
+        margin: '0.75rem',
+      }}
+    >
+      <InfoWidgetModalProvider value={null}>
+        <DeleteTimedTextTrackUploadModalProvider value={null}>
+          <TimedTrackModalWrapper />
+          <FoldableItem
+            infoText={intl.formatMessage(messages.info)}
+            initialOpenValue={true}
+            title={intl.formatMessage(messages.title)}
           >
-            <FormattedMessage {...messages.transcriptSelectPlaceholder}>
-              {(message) => <option value={''}>{message}</option>}
-            </FormattedMessage>
-            {options.map((language) => (
-              <option key={language.value} value={language.value}>
-                {language.label}
-              </option>
-            ))}
-          </select>
-        </form>
-      </Box>
-      {selected.transcript && (
-        <Box>
-          <TranscriptReader
-            transcript={selected.transcript}
-            key={selected.transcript.id}
-          />
-          <Box>
-            <ActionLink
-              margin={'small'}
-              alignSelf="center"
-              color={'status-critical'}
-              label={<FormattedMessage {...messages.hideTranscript} />}
-              onClick={disableTranscript}
-            />
-          </Box>
-        </Box>
-      )}
-    </React.Fragment>
+            <Box
+              direction="row"
+              gap="small"
+              style={{
+                marginTop: '0.75rem',
+                marginBottom: '0.75rem',
+              }}
+            >
+              <Box width="50%">
+                <Select
+                  id="languages"
+                  name="language_choices"
+                  options={options}
+                  replace={false}
+                  labelKey="label"
+                  value={selectedOption.label}
+                  valueKey={{ key: 'value', reduce: true }}
+                  valueLabel={(label) => (
+                    <Box pad="small">
+                      <Text color="blue-active">{`${label}`}</Text>
+                    </Box>
+                  )}
+                  onChange={({ option }) => {
+                    setSelectedOption(option);
+                  }}
+                />
+              </Box>
+              <Box width="50%">
+                <Button
+                  a11yTitle={intl.formatMessage(messages.transcriptDownload)}
+                  color="blue-active"
+                  download
+                  disabled={false}
```

Comment:
This button should be disabled while no transcript is selected. Otherwise we have an active button without any action when we click on it.